mlir/lib/Conversion/TosaToLinalg/TosaToLinalg.cpp

```diff
@@ -2617,3 +2617,37 @@ void mlir::tosa::populateTosaToLinalgConversionPatterns(
       TileConverter>(patterns->getContext());
   // clang-format on
 }
+
+void mlir::tosa::populateTosaToLinalgTypeConversion(TypeConverter &converter) {
```

Comment:
Looks good in general but I'm wondering if a common case type converter could sit as a utility function within the source dialect itself ?

Reply:
Good idea, I moved this to `mlir/lib/Dialect/Tosa/Transforms/TosaTypeConverters.cpp`